hud_src/resource/ui/hudobjectivetimepanel.res

```diff
@@ -35,12 +35,7 @@
 		"visible"		"1"
 		"enabled"		"1"
 		"image"			"../hud/objectives_timepanel_blue_bg"	
-		"scaleImage"		"1"	
-
-		if_match
```

Comment:
Looks like if_match is used for when the tf team hud cvar is enabled, is it safe to remove these? Have you doublechecked these changes work fine with that convar on and off?
https://www.teamfortress.tv/post/870326/hud-editing-short-questions-quick-answers

Reply:
if you enabled or disabled the team status the timer remain the same , it's totally safe to remove it

Reply:
Ah, you've moved all of these such that when team status is affected they don't need to change at all? Nice.